docsv2/bug-reports.md

```diff
@@ -0,0 +1,31 @@
+# Reporting Issues
+
+> [!Important]
+> Change the link in `.github/ISSUE_TEMPLATE/bug-report.yaml` once you change/move the filename.
+
+If you require support with a bug, you can help us help you.
+Please understand that Tasks is mostly maintained by a single person in her free time.
```

Comment:
I would change 'her' to 'their', as a few people make invalid assumptions once they know gender...